src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component } from 'react';
+
+type State = {
+  today: Date,
+};
+
+type Props = {
+  clockName: string,
+};
+
+function formattedDate(date: Date) {
+  return date.toUTCString().slice(-12, -4);
+}
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+
+      window.console.info(
+        formattedDate(this.state.today),
+      );
```

Comment:
don't use state here, it can be outdated
save `new Date()` result to constant and use it for both console.info and setState